doc/docs.md

```diff
@@ -1600,7 +1600,7 @@ type Alphabet = Abc | Xyz
 
 x := Alphabet(Abc{'test'}) // sum type
 if x is Abc {
-	// x is automatically casted to Abc and can be used here
+	// x is automatically cast to Abc and can be used here
```

Comment:
> The verb cast is conventionally uninflected in the past tense and as a past participle. Casted is an old form—examples are easily found in texts from every century from the 14th to the present—but it has given way to cast in modern English. In current usage, however, casted is gaining ground, especially where cast means either (1) to assemble actors for a performance, or (2) to throw out bait and/or a lure on a fishing line. (Both these senses have extended metaphorical uses where casted is likewise used at least some of the time). Many people object to casted, but that doesn’t change the fact that it is catching on and not likely to go away soon.

This form is very often used in programming to avoid confusion. For example:

https://doc.rust-lang.org/reference/items/enumerations.html

[Field-less enums](https://doc.rust-lang.org/reference/items/enumerations.html#field-less-enum) can be casted if they do not have explicit discriminants, or where only unit variants are explicit.

Reply:
Good point, this one it's often not marked as typo by spell checkers.